hs-init.hs

```diff
@@ -154,6 +147,10 @@ generateProject repo owner description Targets{..} = do
             if github
             then decisionToBool ciFlag target
             else falseMessage target
+  privat <- let privTarget = "Private repository" in
+            if github
+            then decisionToBool privateFlag privTarget
+            else falseMessage privTarget
```

Comment:
Looks like there's some code duplication between `ci`, `privat` and `isExe` flag. Maybe this can be refactored?